BuffBot/currency.py

```diff
@@ -1,27 +1,33 @@
 import time
-
+import database
+import datetime
 
 class Currency:
     def __init__(self):
         self.name = 0
 
-    # TODO Make a session persist betweeen voice room changes if not in AFK
+    # TODO Make a session persist between voice room changes if not in AFK
     def register_activity(self, before, after):
 
         if check_if_connected(after) and not member_in_channel("AFK", after):
             print("You're in", after.name)
-            # TODO These database calls does not currently work
-            # database.Database.set_coin_count_session_start(after.id, time.gmtime(), 0, 1)
+            print(time.time())
+            database.Database.set_coin_count_session_start(after.id, time_now(), 0, 1)
         elif not member_in_channel("AFK", after):
             print("You're out", after.name)
-            # database.Database.set_coin_count_session_end(before.id, time.gmtime(), 0)
+            database.Database.set_coin_count_session_end(before.id, time_now(), 0)
 
         # If user is in a channel named AFK, end session. IE: not get coins for being afk
         if member_in_channel("AFK", after):
             print("You're afk")
             # database.Database.set_coin_count_session_end(before.id, time.gmtime(), 0)
```

Comment:
Instead of printing stuff everytime someone is afk, is it just better to remove it?